pages/reference/api/_meta.en-US.json

```diff
@@ -0,0 +1,166 @@
+{
+  "create-a-stream": {
```

Comment:
Will we have to maintain this list? It won't get autogenerated based on the schema?

Reply:
Nope, this list is automatically generated here: https://github.com/livepeer/docs/blob/c43e4854280d69c2c0c822c5bd065616fd9e1461/generate-api-reference.js#LL45C1-L56C2

Reply:
Yes, this list is automatically generated here: https://github.com/livepeer/docs/blob/c43e4854280d69c2c0c822c5bd065616fd9e1461/generate-api-reference.js#LL45C1-L56C2